packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js

```diff
@@ -635,35 +654,45 @@ testModule('API', moduleConfig, () => {
 
         test('converter option runtime change should update html converter', function(assert) {
             const firstConverter = {
-                toHtml: sinon.stub(),
-                fromHtml: sinon.stub(),
+                toHtml: sinon.stub().callsFake((e) => e),
+                fromHtml: sinon.stub().callsFake((e) => e),
             };
 
             const secondConverter = {
-                toHtml: sinon.stub(),
-                fromHtml: sinon.stub(),
+                toHtml: sinon.stub().callsFake((e) => e),
+                fromHtml: sinon.stub().callsFake((e) => e),
             };
 
             const instance = $('#htmlEditor').dxHtmlEditor({
                 converter: firstConverter,
+                value: 'value',
             }).dxHtmlEditor('instance');
 
             instance.option('converter', secondConverter);
-            instance.option('value', 'new value');
 
-            assert.strictEqual(firstConverter.toHtml.callCount, 0);
+            assert.strictEqual(firstConverter.toHtml.callCount, 1);
             assert.strictEqual(firstConverter.fromHtml.callCount, 0);
```

Comment:
we already have assert for "firstConverter" in other tests

Reply:
[174f4dd0ef9c7172f46b44ef322d929e49b24a6b]